app/src/main/java/com/doubtless/doubtless/screens/main/MainActivity.kt

```diff
@@ -52,8 +57,22 @@ class MainActivity : AppCompatActivity(), BackPressDispatcher {
             }
         }
 
-        if (!backPressConsumed)
-            super.onBackPressed()
+        if (!backPressConsumed) {
+            when {
+                doubleBackToExitPressedOnce -> super.onBackPressed()
+                else -> {
+                    this.doubleBackToExitPressedOnce = true
+                    shortToast(R.string.press_again_to_exit)
```

Comment:
I guess snackbar would be a better choice as compared to toast.

Reply:
added snackbar